property_tests/arbitraries/stable_b_tree_map_arb.ts

```diff
@@ -0,0 +1,30 @@
+import fc from 'fast-check';
+import { UniqueIdentifierArb } from './unique_identifier_arb';
+import { createUniquePrimitiveArb } from './unique_primitive_arb';
+import { CandidValueAndMetaArb } from './candid/candid_value_and_meta_arb';
+
+export const StableBTreeMapArb = fc
+    .tuple(
+        CandidValueAndMetaArb(),
+        CandidValueAndMetaArb(),
+        UniqueIdentifierArb('stableBTreeMap'),
+        createUniquePrimitiveArb(
+            fc.nat({
+                max: 254
+            })
+        )
+    )
+    .map(([keySample, valueSample, uniqueIdentifier, memoryId]) => {
+        const name = `stableBTreeMap${uniqueIdentifier}`;
+
+        return {
+            name,
+            body: `let ${name} = StableBTreeMap<${keySample.src.candidTypeAnnotation}, ${valueSample.src.candidTypeAnnotation}>(stableJson, stableJson, ${memoryId});`,
```

Comment:
hmmm, `body` is an interesting name here. When I was reading it later in the StableBTreeMap method tests it read more like it was something else. Isn't this a constructor, or variable declaration? I just wouldn't think of this as a "body" so maybe there's a better name for this.